repository/kafka/docs/latest/monitoring.md

```diff
@@ -0,0 +1,38 @@
+# Monitoring
+
+By default, the KUDO Kafka Operator comes with the JMX Exporter agent enabled. 
+
+When the Kafka operator is deployed with the parameter `METRICS_ENABLED=true` (which defaults to `true`) then:
+
+- Each broker bootstraps with the [JMX Exporter](https://github.com/prometheus/jmx_exporter) java agent exposing the metrics at `9094/metrics`
+- Adds a port named `metrics` to the Kafka Service
+- Adds a label `kubeaddons.mesosphere.io/servicemonitor: "true"` for the service monitor discovery. 
```

Comment:
kind of weird seeing kubeaddons here since it's non-public thing...

Reply:
good catch! it is actually `kudo.dev/servicemonitor: "true"`